sympy/simplify/tests/test_simplify.py

```diff
@@ -468,6 +468,12 @@ def test_logcombine_complex_coeff():
     assert logcombine(i, force=True) == i
     assert logcombine(i + 2*log(x), force=True) == \
         i + log(x**2)
+        
+        
+def test_issue_5950():
+    assert logcombine(log(3) - log(2)) == log(3/2)
```

Comment:
Since `log(3/2)` gives `log(3) - log(2)`, are you sure that you fixed this? Perhaps `assert logcombine(log(3) - log(2)) == log(3/2, evaluate=False)` would be better?

Reply:
In addition, it should be `log(Rational(3, 2), evaluate=False)` (or `log(S(3)/2, evaluate=False)`).